pkg/monitoring/rules/alerts/operator.go

```diff
@@ -75,10 +75,10 @@ func operatorAlerts() []promv1.Rule {
 		},
 		{
 			Alert: "VMStorageClassWarning",
-			Expr:  intstr.FromString("(count(kubevirt_ssp_vm_rbd_block_volume_without_rxbounce > 0) or vector(0)) > 0"),
+			Expr:  intstr.FromString("(count(kubevirt_ssp_vm_rbd_block_volume_without_rxbounce * on(name, namespace) kubevirt_vmi_info{os=\"Microsoft Windows\"} > 0) or vector(0)) > 0"),
```

Comment:
Shouldn't you use `guest_os_name` label?

Reply:
"os" is collected from the annotations, yes, you are right

Reply:
updated @assafad

Reply:
Please consider using both to improve coverage, since not all windows VMs has the guest agent installed.

Reply:
added @assafad